script.js

```diff
@@ -1,35 +1,46 @@
-// 1. Create An Array Of Students.
 let studentsOne = [
-  { name: "Hammad", age: 17 },
-  { name: "Abdul Rafay", age: 11 },
-  { name: "Huzaifa", age: 8 },
-  { name: "Laraib", age: 16 },
+  { Name: "Hammad", Age: 17, Class: "10th" },

```

Comment:
Why you changed then into capital letter? it always should be in small case.

Reply:
I will make sure that it will never happen again and fix it.